test/exitHandler.js

```diff
@@ -147,6 +147,33 @@ contract('ExitHandler', (accounts) => {
         assert(aliceBalanceBefore.add(new BN(50)).eq(aliceBalanceAfter));
       });
 
+      it('Should allow to limbo exit', async () => {
+        const period = await submitNewPeriod(depositTx);
+        const depositProof = period.proof(depositTx);
+        const outputIndex = 1;
+        const exitStake = exitHandler.exitStake();
+        
+        // transaction hash + constant signed by alice
+        const signature = (transferTx + c).sign(alicePriv);
+
+        const inTxData = Tx.parseToParams(transferTx);
+        
+        // Bob will start the exit for Alice
+        await exitHandler.startLimboExit(inTxData, signature, outputIndex, 
```

Comment:
Who will start the limbo exit for tx A->B? A or B?
(Current) B starts a limbo exit by submitting all data A would submit if A were exiting from the coin, as well as the signature received and the in-flight transaction.. All data A would submit will be the depositTx and depositProof.